helpers/vdisk.py

```diff
@@ -144,23 +143,21 @@ def get_config_params(vdisk_name, vpool_name, api, timeout=GET_CONFIG_PARAMS_TIM
             VDiskHelper.LOGGER.info("Setting config vDisk `{0}` should have succeeded".format(vdisk_name))
             return task_result[1]
 
-    @staticmethod
-    def scrub_vdisk(vdisk_guid, api, timeout=15 * 60, wait=True):
+
+    def scrub_vdisk(cls, vdisk_guid, timeout=15 * 60, wait=True):
```

Comment:
Missing classmethod decorator. It will get treated as if is was self (which would never work as we do not instantiate this class)